data/com.solus-project.brisk-menu.gschema.xml

```diff
@@ -26,5 +26,10 @@
       <summary>Search bar position</summary>
       <description>Where the search bar will appear in the menu</description>
     </key>
+    <key type="s" name="hot-key">
+      <default>'&lt;Ctrl&gt;F10'</default>
```

Comment:
I fixed this, makes sense to use CDATA here. As to defaulting to the Super_L key, seems like the most appropriate default, but without having had a gsettings entry for it in the past, it will effectively change the keybinding for all users of Brisk Menu without warning. I'll defer to you on this decision